seqr/views/apis/data_manager_api.py

```diff
@@ -380,15 +397,26 @@ def load_phenotype_prioritization_data(request):
         info.append(f'Project {project_name}: {delete_info}loaded {len(indiv_records)} record(s)')
 
         to_delete |= exist_records
-        all_records += indiv_records
+        all_records_by_project_name[project_name] = indiv_records
 
     if error:
         return create_json_response({'error': error}, status=400)
 
     if to_delete:
         PhenotypePrioritization.bulk_delete(request.user, to_delete)
 
-    PhenotypePrioritization.bulk_create(request.user, [PhenotypePrioritization(**data) for data in all_records])
+    models_to_create = []
+    for indiv_records in all_records_by_project_name.values():
+        for record in indiv_records:
+            model = PhenotypePrioritization(**record)
+            model.guid = f'PP{random.randint(10 ** 8, 10 ** 9)}_{model.individual.individual_id}_{model.gene_id}_{model.disease_id}'[:PhenotypePrioritization.MAX_GUID_SIZE]  # nosec
```

Comment:
`bulk_update` doesn't call model.save() so model._compute_guid() is never called. I found one other example in the repo where random.randint is used to help create the guid during a bulk_create operation.

Reply:
If you wanted to get fancy you could move the guid population to `ModelWithGUID.bulk_create` so theres no need to copy paste this again, but also thats kind of scope creep so this is also okay to leave as is